deploy/README.md

```diff
@@ -284,6 +284,9 @@ REQUIRED_NUMBER_OF_VALIDATORS=1
 # correctly to the Foreign network.
 VALIDATORS=0x 0x 0x
 
+HOME_REWARDABLE=false
```

Comment:
Please add descriptions how it is done for these variables in other sections.

Reply:
Which modes of `REWARDABLE` are supported in this bridge mode?

Reply:
In ERC_TO_ERC mode, for `HOME_REWARDABLE` variable only  `BOTH_DIRECTIONS` is supported. For `FOREIGN_REWARDABLE` variable collecting fees on Foreign network is not supported